canonical/specs-v1.canonical.com
# Added CI checks

## Done
- Added CI checks

## QA
- All checks should pass

Fixes [WD-7059](https://warthogs.atlassian.net/browse/WD-7059?atlOrigin=eyJpIjoiM2FiMmQ5NzI5YWE0NDdkNzlmNGUzMzIxODA1OGNkNmYiLCJwIjoiaiJ9)

[WD-7059]: https://warthogs.atlassian.net/browse/WD-7059?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
